pkg/controllers/trace_controller.go

```diff
@@ -45,7 +45,7 @@ const (
 
 // TraceReconciler reconciles a Trace object
 type TraceReconciler struct {
-	client.Client
+	Client client.Client
```

Comment:
Why has it to be public?

Reply:
It was already accessed outside of the pkg:

gadget-container/gadgettracermanager/controller.go

```
&controllers.TraceReconciler{
		Client:         mgr.GetClient(),
		Scheme:         mgr.GetScheme(),
		Node:           node,
		TraceFactories: traceFactories,
		TracerManager:  tracerManager,
	}
```

I am not sure how implicit fields are working. This code pattern comes from the SDK Operator which generated this code.